src/org/opensolaris/opengrok/search/Results.java

```diff
@@ -253,6 +247,20 @@ public static void prettyPrint(Writer out, SearchHelper sh, int start,
                 out.write("</tt></td></tr>\n");
             }
         }
+        out.write("</tbody>");
```

Comment:
really?
(I am just confused it wasn't there before ... isn't it close somewhere else? )

Reply:
I was surprised as well. It seems the browser adds it there by itself.

Simple test: (when I `Inspect Element` the `<tbody>` is there)
```html
<!doctype html>
<html>
<head>
</head>
<body>
	<table>
		<tr><td>test1</td></td>
		<tr><td>test2</td></td>
	</table>
</body>
</html>
```